### PR TITLE
Sync revert

### PR DESCRIPTION
**Pull request by lenak25**

### CODE REVIEW

File: src/worker/controller/actions/sync/IdentifyMissingStatesAction.js [hunk 1]
```diff
@@ -13,18 +12,19 @@ const NODE_NOTIY = constants.NODE_NOTIFICATIONS;
...
   constructor(controller) {
     this._controller = controller;
   }
-
   execute(params) {
     const useCache = params.cache;
-    const finalCallback = params.onResponse;
+    const callback = params.onResponse;
     if (useCache) {
       this._controller.cache().getAllTips((err, tipsList) => {
```
Comment (assafmo) [inline src/worker/controller/actions/sync/IdentifyMissingStatesAction.js:29]: Can this be promisified?

**Review by assafmo — DISMISSED** [review]
Looks good, thanks.

File: src/worker/controller/actions/sync/IdentifyMissingStatesAction.js (hunk 1 above)
Comment (lenak25) [reply src/worker/controller/actions/sync/IdentifyMissingStatesAction.js:29]: Done

**Comment by lenak25** [comment]
@assafmo please re-review :)

**Review by assafmo — APPROVED** [review]

**Pull request merged**